.rubocop.yml

```diff
@@ -1,29 +1,169 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 AllCops:
+  TargetRubyVersion: 2.1
+  TargetRailsVersion: 3.2
   Include:
     - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
-    - 'spec/**/*'
     - !ruby/regexp /old_and_unused\.rb$/
 
-Documentation:
+# OFN SETTINGS
+# Cop settings that have been agreed upon by the OFN community
+
+Rails:
+  Enabled: true
+
+Style/Documentation:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
+
+# TEMPORARY/CONTESTED SETTINGS
+# These are still to be decided upon, but recommended for inclusion by
+# oeoeaio after scrutinising offenses the codebase
+
+# Don't think this is a big issue, mostly picking up RPSEC scope definitions
+# with lamdas and RSpec '.to change{}' blocks
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
+# No alternative to this one until we upgrade to Rails 4 and can use #find_by
+Rails/DynamicFindBy:
+  Enabled: false
```

Comment:
See my comment on `ClassAndModuleChildren`. 

I just did a test on my machine. I have a rubocop_todo.yml with the following content:

```ruby
Offense count: 378
RSpec/AnyInstance:
  Exclude:
    - some_random_controller_spec.rb
```

I run the following on my terminal:

```shell
$ rubocop some_random_controller_spec.rb
```

and I get no errors. I remove `some_random_controller_spec.rb` from the `rubocop_todo.yml` and execute rubocop again:

```shell
$ rubocop some_random_controller_spec.rb

some_random_controller_spec.rb:178:15: C: Avoid stubbing using expect_any_instance_of.                        
              expect_any_instance_of(Onboarding::Process)             
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^             
some_random_controller_spec.rb:212:15: C: Avoid stubbing using allow_any_instance_of.                         
              allow_any_instance_of(Onboarding::Process).to receive(:user_errors?).and_return(true)                                         
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^              
some_random_controller_spec.rb:228:15: C: Avoid stubbing using expect_any_instance_of.                        
              expect_any_instance_of(AppLink).to receive(:destroy)    
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                         
some_random_controller_spec.rb:236:15: C: Avoid stubbing using allow_any_instance_of.                         
              allow_any_instance_of(Onboarding::Process).to receive(:app_link_errors?) { true }                                             
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^              
some_random_controller_spec.rb:253:15: C: Avoid stubbing using expect_any_instance_of.                        
              expect_any_instance_of(AppLink).to receive(:destroy)    
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                         

1 file inspected, 7 offenses detected            
```